foodbot.rb

```diff
@@ -0,0 +1,35 @@
+require 'httparty'
+require 'parallel'
+require_relative 'bon_apetit'
+
+class Foodbot
+
+  SLACK_HOOK_URL = ENV['SLACK_HOOK_URL']
+
+  def initialize(restaurants)
+    @restaurants = restaurants
+    @commands = restaurants.keys + ["all"]
+  end
+
+  def respond_to(command)
+    if command == "all"
+      Parallel.map(@restaurants.values) {|res| res.menu_text }
+              .push("\n_#{BonApetit.random}_").join("\n")
+              .tap {|text| post_to_slack(text)}
```

Comment:
Please return only the bon apetit phrase her. In case of calling foodbot with 'all' argument, the one who called it gets the output twice: once from the post integration and one (seen only by the caller) from the slackbot.